core/src/Cuda/Kokkos_Cuda_Parallel.hpp

```diff
@@ -1110,6 +1158,17 @@ class ParallelReduce<FunctorType, Kokkos::RangePolicy<Traits...>, ReducerType,
                                  Policy::is_graph_kernel::value ||
 #endif
                                  !std::is_same<ReducerType, InvalidType>::value;
+
+#ifdef KOKKOS_THRUST
```

Comment:
this should KOKKOS_ENABLE_THRUST likely but we can rename when this works. We may wanna just define it for this draft PR so you can see whether this passes testing.